lib/gems/pending/appliance_console/cli.rb

```diff
@@ -145,6 +151,7 @@ def run
       install_ipa if install_ipa?
       install_certs if certs?
       extauth_opts if extauth_opts?
+      server_mode if server_state?
```

Comment:
Generally, I think this type of change throughout this file would make it a lot easier to read, but let's stick to this change for this patch.